packages/bodiless-richtext/src/RichText.tsx

```diff
@@ -231,6 +233,7 @@ const BasicRichText = React.memo(<P extends object>(props: P & RichTextProps) =>
       finalComponents: finalComponents$,
       plugins: getPlugins(finalComponents$),
       globalButtons: getGlobalButtons(finalComponents$),
+      deserializers: getDeserializers(finalComponents$),
```

Comment:
do we need this deserializers here?

Reply:
nope, it is not needed here
we can remove it